docs/docs/cmd/spo/tenant/tenant-settings-set.md

```diff
@@ -265,7 +265,7 @@ m365 spo tenant settings set [options]
 : Syncs Azure B2B Management Policies. Allowed values `true,false`. For more information, see [SharePoint and OneDrive integration with Azure AD B2B](https://aka.ms/spo-b2b-integration).
 
 !!! important
-    To use this command you have to have permissions to access the tenant admin site.
+    To use this command you have to have permission to access the tenant admin site.
```

Comment:
Let's put this under a `Remarks´ header like we do for other commands. Also I notice that the default options are on top for this command instead of being at the bottom.

Reply:
There are a bunch of other commands that also include this under the heading options. e.g. `spo term group add`, `spo theme list`, `spo homesite set`, ...

I definitely agree that this should be placed under the heading remarks but let's create an issue for this. I'll create a new one shortly.

Reply:
Nevertheless, could you move the default options to the bottom of the section?